pkgs/development/haskell-modules/configuration-ghc-9.6.x.nix

```diff
@@ -193,6 +196,9 @@ in
 
   # A given major version of ghc-exactprint only supports one version of GHC.
   ghc-exactprint = addBuildDepend self.extra super.ghc-exactprint_1_7_1_0;
+
+  ghc-lib-parser = doDistribute self.ghc-lib-parser_9_10_2_20250515;
```

Comment:
Not yet sure what to do with the ghc-lib* packages, hence I haven't really touched it. We'll probably have to decide based on whether things (HLS, formatters, hlint, …) build with this version or another one.